JS/contactForm.js

```diff
@@ -2,22 +2,42 @@
 
 // the input valdiation
 const contactForm = document.querySelector('.inputForm');
+
 const emailPattern = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+\.[a-z]{2,3}/;
 
+const UserName = document.getElementById('fullName');
+const UserEmail = document.getElementById('email-input');
+const UserMessage = document.getElementById('textarea-input');
+
 contactForm.addEventListener('submit', (event) => {
-  const UserInput = document.getElementById('email-input');
-  const vaild = emailPattern.test(UserInput.value);
+  const vaild = emailPattern.test(UserEmail.value);
   const errorMsg = document.getElementById('errorMessage');
-
   if (!vaild) {
-    errorMsg.style.height = '20px';
+    errorMsg.style.height = '40px';
+    errorMsg.style.width = 'fit-content';
     errorMsg.style.background = '#FFD600';
     errorMsg.style.borderRadius = '5px';
     errorMsg.style.alignSelf = 'center';
-    errorMsg.style.padding = '5px';
+    errorMsg.style.padding = '6px';
     errorMsg.innerHTML = '*Email must be in email format (abd@sdm.com) and in lowercase letters!';
     event.preventDefault();
-    UserInput.value = '';
-    UserInput.focus();
   }
+  return false;
 });
+
+contactForm.addEventListener('submit', (event) => {
+  const formUserInput = new FormData(event.target);
+  const UserInput = {};
+  formUserInput.forEach((value, key) => {
+    UserInput[key] = value;
+  });
+  const storedData = JSON.stringify(UserInput);
+  localStorage.setItem('UserData', storedData);
```

Comment:
- [ ] 🙏Please ensure that the form data is stored in the local storage when the user👨‍👩‍👧‍👦 inputs data without clicking the submit button. This will help users of your app to avoid losing data when they close the page.